### DISCUSSION TITLE
Application approved, but not receive any confirmation mail.

### DISCUSSION THREAD

**Discussion by safwanahmadsaffi** [community/community#147083] [GitHub Education]
### Hi everyone,

I received this message on my profile but did not receive any mail. Congrats! Your request for [@safwaahmad](https://github.com/safwaahmad) was approved and your benefits will be available [soon](https://github.com/orgs/community/discussions/111352#user-content-how-long-after-ive-been-approved-will-i-receive-my-academic-benefits). How to get free credentials?

**Comment by BryanLomerio** [comment]
That message seems suspicious—don’t click links or share info. Verify the sender, check your email for related messages, and contact the platform's support for clarification. If it’s a scam, report it and ignore it.

**Comment by davevad93** [comment]
Hi @safwaahmad ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @safwaahmad was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640, #145995

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
